src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/LegalEntitySignedAgreementViewModel.cs

```diff
@@ -1,19 +1,13 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
+﻿namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
 {
     public class LegalEntitySignedAgreementViewModel
     {
-        public string HashedAccountId { get; set; }
+        public string AccountHashedId { get; set; }
         public string TransferConnectionCode { get; set; }
         public string LegalEntityCode { get; set; }
```

Comment:
what is legal entity code?

Reply:
Legal Entity Code  comes from  [employer_account].LegalEntity table